frameworks/mlr3automl/exec.R

```diff
@@ -0,0 +1,74 @@
+library(mlr3)
+library(mlr3learners)
+library(mlr3extralearners)
+library(mlr3tuning)
+library(mlr3pipelines)
+library(mlr3automl)
+library(mlr3oml)
+
+run <- function(train_file, test_file, target.index, type, output_predictions_file, cores, time.budget, seed, name) {
+  start_time = Sys.time()
+  # AutoML benchmark uses unsigned 32bit integers as seeds, which may be too large for R (32bit signed)
+  if (seed > .Machine$integer.max) {
+    set.seed(42)
+  } else {
+    set.seed(seed)
+  }
+  train <- mlr3oml::read_arff(train_file)
+  colnames(train) <- make.names(colnames(train))
+  target <- colnames(train)[target.index]
+
+  test <- mlr3oml::read_arff(test_file)
+  colnames(test) <- make.names(colnames(test))
+  
+  preprocessing = "stability"
```

Comment:
The benchmark requires that there be an "all-defaults" version of the framework submitted.  I think most of these values are the same as the defaults, but if you can remove these manual settings and leave the defaults as-is (unset in the `AutoML()` function), then it's easier to look at the code and see that no non-default settings are activated (helpful for review and also for reproducibility).

We have seen a few frameworks submit a default version and also a ["compete" mode](https://github.com/openml/automlbenchmark/blob/master/resources/frameworks.yaml#L21-L31) version (where at most one non-default param value is set), but we mark those non-default frameworks differently, since they are not fully "auto".  So if you wanted to submit "two" frameworks (one with the preset option `preprocessing = "stability"`, you could do that, but budget-wise, we don't have resources to benchmark both, so it's more straight-forward and more true to the "automatic" nature of an AutoML system, if you submit the default one.